canonical/ubuntu.com
# Fix containers forms and remove unused forms

## Done
- Update /containers#get-in-touch dynamic form design and logic - consistent with other dynamic forms. Fixes 500 on submission.
- Remove unused `kernel-form.js`

## QA

- Go to https://ubuntu-com-14183.demos.haus/containers#get-in-touch
  - Submit form and check payload is correct
- Go to prod: https://ubuntu.com/internet-of-things#beta-signup
  - Go to demo: https://ubuntu-com-14183.demos.haus/internet-of-things#beta-signup
  - See that nothing happens for both prod and demo, the triggering hash does not invoke the modal
  - The triggering hash is only called in one line throughout the entire file, and it redirects users to /iot#beta-signup page.

## Issue / Card

Fixes [WD-14054](https://warthogs.atlassian.net/browse/WD-14054)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-14054]: https://warthogs.atlassian.net/browse/WD-14054?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
